app-modules/prospect/src/Filament/Resources/ProspectResource/Pages/ManageProspectCareTeam.php

```diff
@@ -84,7 +84,7 @@ public function table(Table $table): Table
             ])
             ->headerActions([
                 AttachAction::make()
-                    ->label('Add to Care Team')
```

Comment:
Let's reference the translation here so that it would be updated with the rest of the default actions.